Source/WebKit/UIProcess/WebsiteData/WebsiteDataStore.cpp

```diff
@@ -378,6 +378,12 @@ void WebsiteDataStore::resolveDirectoriesIfNecessary()
         auto resolvedCookieDirectory = resolveAndCreateReadWriteDirectoryForSandboxExtension(FileSystem::parentPath(m_configuration->cookieStorageFile()));
         m_resolvedConfiguration->setCookieStorageFile(FileSystem::pathByAppendingComponent(resolvedCookieDirectory, FileSystem::pathFileName(m_configuration->cookieStorageFile())));
     }
+#if USE(CURL)
+    if (!m_configuration->alternativeServicesStorageFile().isEmpty()) {
```

Comment:
We already have this above:
if (!m_configuration->alternativeServicesDirectory().isEmpty())
Can't we just use that?

Reply:
Fixed